x-pack/legacy/plugins/apm/public/components/app/ServiceMap/MapOfServices.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import CytoscapeComponent from 'react-cytoscapejs';
+import { IUrlParams } from '../../../context/UrlParamsContext/types';
+
+function elementId(serviceName: string, environment?: string) {
+  return serviceName + '/' + (environment ? '/' + environment : '');
+}
+
+function label(serviceName: string, environment?: string) {
+  return serviceName + (environment ? '/' + environment : '');
+}
+
+// interface Props {
+//   connections: any;
```

Comment:
Without having looked into it much I don't think the Graph plugin is easily embeddable in APM UI. Afaict Graph is built in Angular while APM UI is React. Mixing those frameworks is possible but something we should think twice about doing.

Secondly, I think our use case requires a lot of custom UI work and we would probably spend more time fighting the capabilities and visual appearance of Graph instead of building it from scratch. We already had a meeting with other teams who also need graph visualizations and we found common ground in using the same lib (`cytoscape`) but for now nothing more than that.